quick_sort/quick_sort.py

```diff
@@ -7,17 +7,13 @@ def median(a, i, j, k):
     :param k: middle element index
     :return: return median of values at indices i, j and k.
     """
-    if a[i] > a[j]:
-        if a[j] > a[k]:
-            return j
-        else:
-            return k
-    else:
-        if a[i] > a[k]:
-            return i
-        else:
-            return k
-
+    ai, aj, ak = a[i], a[j], a[k]
+    med_val = ai + aj + ak - max(ai, aj, ak) - min(ai, aj, ak)
+    if ai == med_val:
+        return i
+    elif aj == med_val:
+        return j
+    return k
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `pep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp4__xq2to/quick_sort/quick_sort.py
+++ b/tmp/tmp4__xq2to/quick_sort/quick_sort.py
@@ -14,6 +14,7 @@
     elif aj == med_val:
         return j
     return k
+
 
 def partition(array, l, r):
     """
```

Reply:
Ah yes - my apologies. My pep8 vim plugin did not catch this for some reason.... updating the PR now!